### ISSUE TITLE
Docker: allowed directories error

### ISSUE THREAD

**Issue by mrx23dot** [ConsenSys/mythril#1579]
I'm running docker example from documentation
https://mythril-classic.readthedocs.io/en/master/installation.html
`docker run -v $(pwd):/tmp mythril/myth analyze /tmp/test.sol`

gives error:
```
mythril.interfaces.cli [ERROR]: Solc experienced a fatal error.
Cannot import url ("/tmp/test.sol"): File outside of allowed directories.

source say:
p = Popen([solc_binary, "--bin-runtime", '--allow-paths', ".", file], stdout=PIPE, stderr=PIPE)
but looks like current dir is not /tmp/

```
`docker run -v $(pwd):/tmp mythril/myth version`
Mythril version v0.22.35
Debian 10

**Comment by mrx23dot** [comment]
Why can't we allow  --allow-paths * and make life easier?
mythril is already highly specialised sw for experts.

**Comment by mrx23dot** [comment]
Also applies to non-docker version.
When using latest solc it allows files in: /home/mythril 
But when specifying  --solv v0.4.11 it says:  File outside of allowed directories for same dir.

This is a big mess.

**Comment by norhh** [comment]
@mrx23dot, regarding using older solc versions, versions older than 0.4.20 are not well supported by the py-solc library, hence the error. You can try using v0.4.20 for such cases.

**Comment by mrx23dot** [comment]
No specifying --allow-paths would allow everything. (same as --allow-paths "",/,* )

**Comment by norhh** [comment]
`No specifying --allow-paths would allow everything. (same as --allow-paths "",/,* )`
I was answering about the issue pertaining to versions less than v0.4.18 + py-solc library. They are buggy due to some issues with py-solc. The docker issue is independent of issues pertaining to versions <= 0.4.18. Since that's the reason for why a file in the same directory cannot be recognized by solidity.

**Comment by mrx23dot** [comment]
Solc installed from snap repo had issues finding sources, so I uninstalled it.
When I download the solc binaries manually they work great, always see sources.
Without a solc in PATH myth is not able to download the specified version and use it:

myth analyze simple.sol --solv v0.4.26

```
  File "/root/.local/lib/python3.7/site-packages/solc/wrapper.py", line 159, in solc_wrapper
    stderr=subprocess.PIPE)
  File "/usr/lib/python3.7/subprocess.py", line 775, in __init__
    restore_signals, start_new_session)
  File "/usr/lib/python3.7/subprocess.py", line 1522, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'solc': 'sol
```

Is it possible to manually input the output of the compiler (json/bin/ast?), thus bypassing compiler call?

**Comment by norhh** [comment]
@mrx23dot , you can directly give it a bytecode using `-c` option (default assumes a contract creation code). You need to explicitly add the `--bin-runtime` option when running the runtime bytecode.

**Comment by norhh** [comment]
Should be fixed by #1584
